xml/article_setup.xml

```diff
@@ -1300,16 +1315,18 @@ rmdir /dev/cpuset/$1
      <para>
       The <emphasis>Best Effort</emphasis> scheduler is the default I/O
       scheduler, and is used for all processes that do not specify a different
-      I/O scheduler class. By default, this scheduler sets its nice level
-      according to the nice value of the running process.
+      I/O scheduler class. By default, this scheduler sets its
+      <literal>nice</literal> level according to the nice value of the running
+      process.
      </para>
      <para>
-      There are eight different nice levels available for this scheduler. The
-      lowest priority is represented by a nice level of seven, the highest
-      priority is zero.
+      There are eight different <literal>nice</literal> levels available for
+      this scheduler. The lowest priority is represented by a
+      <literal>nice</literal> level of <literal>7</literal>, and the highest
+      priority is <literal>0</literal>.
      </para>
      <para>
-      This scheduler has the scheduling class number 2.
+      This scheduler has the scheduling class number two.
```

Comment:
I am extremely unsure about this one. I think you should keep the number here too and maybe use a literal. But I really don't know.